tools/proofers/greek2ascii.php

```diff
@@ -90,7 +90,14 @@ function addChar(myChar) {
 
 <tr>
 <td colspan="3">
-<?php echo sprintf( _("Please read the Project Gutenberg <a href='%s' target='_new'>Greek HOWTO</a> for more information."), $PG_greek_howto_url) ; ?> 
+<?php $url = get_faq_url('transliterating-greek');
+
+// In case get_faq_url fails to find the url, default to the Gutenberg page
+if ($url === "")
+    echo sprintf( _("Please read the Project Gutenberg <a href='%s' target='_new'>Greek HOWTO</a> for more information."), $PG_greek_howto_url);
+else
+    echo sprintf( _("Please read the DP Official Documentation <a href='%s' target='_new'>Transliterating Greek</a> for more information."), $url);
```

Comment:
While the url currently used for the href is part of the DP Official Documentation, another site could update the URL in `pinc/faq.inc` to point somewhere else and this phrase would then be wrong. Consider changing the string to read: `Please read the <a href='%s' target='_new'>Transliterating Greek</a> documentation for more information.`